clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/description/manifest/KubernetesManifestAnnotater.java

```diff
@@ -299,4 +299,8 @@ public static KubernetesManifest getLastAppliedConfiguration(KubernetesManifest
         new TypeReference<KubernetesManifest>() {},
         null);
   }
+
+  public static String getManifestCluster(KubernetesManifest manifest) {
+    return manifest.getAnnotations().get(CLUSTER);
+  }
```

Comment:
This function should probably either be annotated `@Nullable` or do a `Strings.nullToEmpty` on the result.

Reply:
👍 👍 Good call!